src/models/userModel.ts

```diff
@@ -47,14 +50,12 @@ const userSchema = new Schema({
   passwordUpdatedAt: Date,
   desc: String,
   img: String,
-  createdAt: {
```

Comment:
We have to handle this ourselves because we have used "models" to define a custom method for our userSchema. timestamps option won't work again.